lib/organisms/widgets-area/widgets-area.directive.js

```diff
@@ -21,12 +21,14 @@ function lnOWidgetsArea($compile) {
     elem.empty();
 
     angular.forEach(scope.lnWidgets, function(widget, index) {
-      var widgetDirective = '<section class="' + widget.type + '" ' + widget.type; 
+      var type = getWidgetType(widget.type);
+      var attrPrefix = (type === widget.type) ? '' : 'ln-';
```

Comment:
Can we come up with a more consistent way of ding this, eg always use the ln- prefix for all attributes (not just widgets).